loader/loader.c

```diff
@@ -4421,6 +4421,9 @@ VkResult loader_create_instance_chain(const VkInstanceCreateInfo *pCreateInfo, c
 
     memcpy(&loader_create_info, pCreateInfo, sizeof(VkInstanceCreateInfo));
 
+    inst->enabled_layer_count = pCreateInfo->enabledLayerCount;
+    inst->enabled_layer_names = pCreateInfo->ppEnabledLayerNames;
```

Comment:
You must copy the data if you intend to use it after returning from vkCreateInstance.

Reply:
I updated the code, I am really not quite sure about this, did I use the right allocator, did I used it correct, did I deleted it at the right place, could loader_create_instance_chain because with the same `loader_instance` object?